web/cobrands/fixmystreet/fixmystreet.js

```diff
@@ -179,6 +179,40 @@ fixmystreet.resize_to = {
   }
 };
 
+fixmystreet.geolocate = {
+    setup: function(success_callback) {
+        $('#geolocate_link').click(function(e) {
+            var $link = $(this);
+            e.preventDefault();
+            // Spinny thing!
+            if($('.mobile').length){
```

Comment:
Since we’ve taken the plunge and moved this chunk of code from its original location, might we also clean up the spacing around these the brackets in this `if`/`else` block?